src/Integration/FrameContextifierIntegration.php

```diff
@@ -78,11 +78,12 @@ public function setupOnce(): void
     private function addContextToStacktraceFrames(int $maxContextLines, Stacktrace $stacktrace): void
     {
         foreach ($stacktrace->getFrames() as $frame) {
-            if ($frame->isInternal() || null === $frame->getAbsoluteFilePath()) {
+            if ($frame->isInternal()) {
                 continue;
             }
 
-            $sourceCodeExcerpt = $this->getSourceCodeExcerpt($maxContextLines, $frame->getAbsoluteFilePath(), $frame->getLine());
```

Comment:
Shouldn't we revert every change in this file now that we always set the `$absoluteFilePath` field except when the frame is internal?